packages/core/src/shared/utilities/pathFind.ts

```diff
@@ -6,44 +6,17 @@
 import * as vscode from 'vscode'
 import * as path from 'path'
 import fs from '../../shared/fs/fs'
-import { ChildProcess, ChildProcessOptions } from './processUtils'
 import { GitExtension } from '../extensions/git'
 import { Settings } from '../settings'
-import { getLogger } from '../logger/logger'
+import { ChildProcess, ChildProcessOptions } from './processUtils'
+import { getLogger } from '..'
 
 /** Full path to VSCode CLI. */
 let vscPath: string
 let sshPath: string
 let gitPath: string
 let bashPath: string
 
-/**
- * Tries to execute a program at path `p` with the given args and
- * optionally checks the output for `expected`.
- *
- * @param p path to a program to execute
- * @param args program args
- * @param doLog log failures
- * @param expected output must contain this string
- */
-export async function tryRun(
```

Comment:
can the function stay in this location? that will help avoid merge conflicts with incoming feature-branches .